scripts/exregional_run_fcst.sh

```diff
@@ -492,6 +492,9 @@ if [ "${RUN_ENVIR}" = "nco" ]; then
     ln_vrfy -sf "${DATA_SHARED}/${NET}.${cycle}${dot_ensmem}.dyn.f${fhr}${mnts_secs_str}.nc" "dynf${fhr}${mnts_secs_str}.nc"
     ln_vrfy -sf "${DATA_SHARED}/${NET}.${cycle}${dot_ensmem}.phy.f${fhr}${mnts_secs_str}.nc" "phyf${fhr}${mnts_secs_str}.nc"
   done
+
+  # create an intermediate symlink to RESTART
+  ln_vrfy -sf "${DATA}/RESTART" "${COMIN}/RESTART"
```

Comment:
Why is a symlink needed in the COMIN directory now? I think the idea in NCO mode is to keep temporary stuff in DATA so I wonder if this is avoidable.